code/game/objects/items/RCD.dm

```diff
@@ -491,6 +504,27 @@ RLD
 					playsound(src.loc, 'sound/machines/click.ogg', 50, 1)
 					return TRUE
 	qdel(rcd_effect)
+	return FALSE
```

Comment:
I don't think this is a required addition, exiting the proc with no value has the same effect